app/models/project_mediaflux.rb

```diff
@@ -2,32 +2,49 @@
 class ProjectMediaflux
   def self.create!(project:, session_id:, created_by:)
     store_name = Store.default(session_id: session_id).name
-    project_name = safe_name(project.directory)
-    project_namespace = "#{Rails.configuration.mediaflux['api_root_ns']}/#{project_name}-ns"
-    tigerdata_values = project_values(project, created_by)
 
     # Create a namespace for the project
     # The namespace is directly under our root namespace
+    project_name = safe_name(project.directory)
+    project_namespace = "#{Rails.configuration.mediaflux['api_root_ns']}/#{project_name}-ns"
     Mediaflux::Http::NamespaceCreateRequest.new(namespace: project_namespace, description: "Namespace for project #{project.title}", store: store_name, session_token: session_id).resolve
 
     # Create a collection asset under the new namespace and set its metadata
+    tigerdata_values = project_values(project: project, created_by: created_by)
     create_request = Mediaflux::Http::CreateAssetRequest.new(session_token: session_id, namespace: project_namespace, name: project_name, tigerdata_values: tigerdata_values)
     create_request.resolve
     create_request.id
   end
 
-  def self.project_values(project, created_by)
-    {
+  def self.update(project:, session_id:, updated_by:)
+    tigerdata_values = project_values(project: project, updated_by: updated_by)
+    Mediaflux::Http::UpdateAssetRequest.new(session_token: session_id, id: project.mediaflux_id, tigerdata_values: tigerdata_values).resolve
+  end
```

Comment:
This is really the main part of this PR. Issue an UpdateAssetRequest (rather than NamespaceCreateRequest and CreateAssetRequest) when updating an existing record.